packages/common/src/interfaces/columnPicker.interface.ts

```diff
@@ -20,6 +20,9 @@ export interface ColumnPickerOption {
   /** Same as "columnTitle", except that it's a translation key which can be used on page load and/or when switching locale */
   columnTitleKey?: string;
 
+  /** Custom sort function for the columns in the grid menu */
```

Comment:
oops super nitpicky but there's another typo here, this is the Column Picker interface and you're mentioning Grid Menu. 😉

Reply:
all good thanks :)